kernelci.org/content/en/kernel-community/_index.md

```diff
@@ -1,19 +1,53 @@
 
-> This documentation is still a **WIP**. As we ready new KernelCI for more users the documentation is being improved and expanded.
+> We know that the documentation above may not answer all your questions. We are working to improve it. We ask you to reach out to our mailing list at [kernelci@lists.linux.dev](mailto:kernelci@lists.linux.dev)  with questions and feedback. We are eager to hear from you!
+
+When using KernelCI, the first step is to configure trees, kernel configs and tests. Then, with the tests running, users want to evaluate results through the [dashboard](https://dashboard.kernelci.org/), [kci-dev](https://kci.dev/) and setup notifications.
+
+## Enabling trees and tests
+
+### Check if KernelCI already runs what you need
+
+Before we learn how to add a tree and/or tests to KernelCI, we should check if that tree is already being tested by the KernelCI ecosystem and which tests are running on it. To do that navigate through the [dashboard](https://dashboard.kernelci.org/) and look for the tree you are interested in. Multiple CI systems (origins) send test results to KernelCI, so remember to check all different origins in the Dashboard. There you can search for specific tests being run.
+
+### Add new trees and tests
+
+If you look at the [KernelCI Architecture](../architecture/), you see **Maestro** and the systems in the **CI ecosystem** contributing results to KCIDB. This guide will explain how to add trees and tests to [Maestro](../maestro/) as it is part of the Core KernelCI infrastructure we make available for the community. If you need to add your tests to another CI system, contact them directly.
 
 This section of the documentation shares instructions to:
 * [enable kernel testing for your tree/branch](../maestro/pipeline/developer-documentation/#enabling-a-new-kernel-tree)
 * [enable specific tests](../maestro/pipeline/developer-documentation/#enabling-a-new-test)
-* [visualize the results](view-results)
 
 For suggestions of device tests, see [device kselftests](device-tests).
 
-We know that the documentation above may not answer all your questions. We are working to improve it. We ask you to reach out to our mailing list at [kernelci@lists.linux.dev](mailto:kernelci@lists.linux.dev)  with questions and feedback. We are eager to hear from you!
+## Interacting with the results
+
+The Kernel Community has a few options to interact with the test results available in KernelCI. Remember this data includes results from several CI systems.
+
+### Web Dashboard
+
+Our [Web Dashboard](https://dashboard.kernelci.org/): The new Dashboard aims at providing an easy way for the community to look at the test results. It is still under development and we are open to your [feedback and feature requests](https://github.com/kernelci/dashboard).
```

Comment:
For the feedback and feature requests should we link to https://github.com/kernelci/dashboard/issues?